apps/src/templates/amazonFutureEngineerEligibility/amazonFutureEngineerEligibilityForm.jsx

```diff
@@ -311,80 +312,72 @@ export default class AmazonFutureEngineerEligibilityForm extends React.Component
+    <option key={state} value={state}>
+      {state}
+    </option>
+  ));
 
-  handleChange = change => {
-    this.props.handleChange(change);
-  };
-
-  render() {
-    // TO DO: Maybe outermost element should be FormGroup, not div
-    return (
+  return (
+    <div>
       <div>
-        <div>
-          Since you checked the box above, please verify your school address
-          below.
-        </div>
-        <FieldGroup
-          id="street1"
-          label="Street 1"
-          type="text"
-          required={true}
-          onChange={this.handleChange}
-          validationState={
-            this.props.checkValidationState('street1')
-              ? VALIDATION_STATE_ERROR
-              : null
-          }
-        />
-        <FieldGroup
-          id="street2"
-          label="Street 2"
-          type="text"
-          required={false}
-          onChange={this.handleChange}
-        />
-        <FieldGroup
-          id="city"
-          label="City"
-          type="text"
-          required={true}
-          onChange={this.handleChange}
-          validationState={
-            this.props.checkValidationState('city')
-              ? VALIDATION_STATE_ERROR
-              : null
-          }
-        />
-        <FieldGroup
-          id="state"
-          label="State"
-          type="text"
-          required={true}
-          onChange={this.handleChange}
-          validationState={
-            this.props.checkValidationState('state')
-              ? VALIDATION_STATE_ERROR
-              : null
-          }
-        />
-        <FieldGroup
-          id="zip"
-          label="Zip code"
-          type="number"
-          required={true}
-          onChange={this.handleChange}
-          validationState={
-            this.props.checkValidationState('zip')
-              ? VALIDATION_STATE_ERROR
-              : null
-          }
-        />
+        Since you checked the box above, please verify your school address
+        below.
       </div>
-    );
-  }
-}
+      <FieldGroup
+        id="street1"
+        label="Street 1"
+        type="text"
+        required={true}
+        onChange={handleChange}
+        validationState={
+          checkValidationState('street1') ? VALIDATION_STATE_ERROR : null
+        }
+      />
+      <FieldGroup
+        id="street2"
+        label="Street 2"
+        type="text"
+        required={false}
+        onChange={handleChange}
+      />
+      <FieldGroup
+        id="city"
+        label="City"
+        type="text"
+        required={true}
+        onChange={handleChange}
+        validationState={
+          checkValidationState('city') ? VALIDATION_STATE_ERROR : null
+        }
+      />
+      <FieldGroup
+        id="state"
+        label="State"
+        required={true}
+        onChange={handleChange}
+        validationState={
+          checkValidationState('state') ? VALIDATION_STATE_ERROR : null
+        }
+        componentClass="select"
```

Comment:
This line and the line below represent the logical changes to this component (the rest is just moving around existing code). To be honest, I do not totally understand this `componentClass` prop, but it is used in [an example here](https://react-bootstrap-v3.netlify.app/components/forms/) and seems to do what I want it to. There are other ways we implement a "select a state" throughout our code, but I was happy to be able to re-use the same component as we use for other fields in this form.